Source/Core/DolphinQt/MainWindow.cpp

```diff
@@ -1284,8 +1284,10 @@ void MainWindow::ShowGraphicsWindow()
               "display", windowHandle())),
           winId());
     }
-#endif
+    m_graphics_window = new GraphicsWindow(m_xrr_config.get(), this);
+#else
     m_graphics_window = new GraphicsWindow(this);
+#endif
```

Comment:
```cpp
    m_graphics_window = new GraphicsWindow(m_xrr_config.get(), this);
#else
    m_graphics_window = new GraphicsWindow(this);
#endif
```

Revert back to
```cpp
#endif

    m_graphics_window = new GraphicsWindow(this);
```

Reply:
I just changed it, could you check to make sure it's correct?